packages/python/instrumentation-sdk/tests/unit/features/spans/test_reliable_reporter.py

```
@@ -0,0 +1,266 @@
+import os
+import uuid
+import pytest
+from datetime import datetime, timezone, timedelta
```

Flake8 (7.3.0)

[error] 'datetime.timedelta' imported but unused

(F401)